### ISSUE TITLE
The unit tests do not clean up after themselves

### ISSUE THREAD

**Issue by eharris** [airsonic/airsonic#1182]
## Problem description

The unit tests leave testing artifacts in `/tmp/airsonic_test_*`.  They should be cleaning up after themselves.

### Steps to reproduce

1. Run a build that includes the unit tests.
2. Observe that artifacts are left in `/tmp`

## System information

 * **Airsonic version**: Current master
 * **Operating system**: Debian stretch amd64

**Comment by tesshucom** [comment]
It is because it has an aspect as evidence.
As it is necessary for back-end development to scrutinize content after test execution.
If we think it is necessary, it is feasible by enabling to switch by profile option.

**Comment by jvoisin** [comment]
I don't think that anyone cares about generated test data if everything went fine.
